plugins/right-sizer/src/controller/controller.go

```diff
@@ -133,7 +133,7 @@ func NewController(stop chan struct{}, kubeClientResources util.KubeClientResour
 		o(cfg)
 	}
 
-	missingAllowedNamespaces := funk.RightJoinString(cfg.allowedNamespaces, cfg.allowedUpdateNamespaces)
+	_, missingAllowedNamespaces := lo.Difference(cfg.allowedNamespaces, cfg.allowedUpdateNamespaces)
```

Comment:
I was a bit worried about this one, but they look functionally the same https://go.dev/play/p/dyaPZ-4Pgv2